### PR TITLE
Fix default test suite by using sqlite3 as the database

### PR DESCRIPTION
**Pull request by doublej472**
I'm not sure if the haystack recipe is required, but I'm trying to replicate the production runlist so it's included here anyways.

### CODE REVIEW

**Comment by jldugger** [comment]
Can you make this a second test suite?

File: attributes/default.rb [hunk 1]
```diff
@@ -27,7 +27,7 @@
 
 default['ganeti_webmgr']['haystack_whoosh_path'] = '/opt/ganeti_webmgr/whoosh_index'
 
-default['ganeti_webmgr']['database']['engine'] = nil
+default['ganeti_webmgr']['database']['engine'] = 'django.db.backends.sqlite3'
```
Comment (jldugger) [inline attributes/default.rb]: I'm thinking this would be better set in some sort of test suite metadata.

**Comment by doublej472** [comment]
!bump patch ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v0.5.1.  Have a nice day!

**Pull request merged**